<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWidgetIcons</class>
 <widget class="QWidget" name="EditWidgetIcons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="defaultIconsRadio">
     <property name="text">
      <string>Use default icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="defaultIconsView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="customIconsRadio">
     <property name="text">
      <string>Use custom icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="customIconsView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add custom icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete custom icon</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defaultIconsRadio</tabstop>
  <tabstop>defaultIconsView</tabstop>
  <tabstop>customIconsRadio</tabstop>
  <tabstop>customIconsView</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
